src/python/doc/alpha_complex_user.rst

```diff
@@ -89,25 +89,28 @@ In order to build the alpha complex, first, a Simplex tree is built from the cel
 Filtration value computation algorithm
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-  **for** i : dimension :math:`\rightarrow` 0 **do**
-    **for all** :math:`\sigma` of dimension i
-      **if** filtration(:math:`\sigma`) is NaN **then**
-        filtration(:math:`\sigma`) = :math:`\alpha^2(\sigma)`
-      **end if**
+.. code-block:: bash
```

Comment:
Looking for some random language that has do, end, else, for, if, then as keywords? I think bash is missing "end" from this list.

Reply:
I had a look to [available lexers](https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/Codeblocks.html#available-lexers). `vim` seems to be a better candidate.